compiler/damlc/daml-stdlib-src/DA/Time.daml

```diff
@@ -58,6 +60,14 @@ minutes x = seconds (60 * x)
 seconds : Int -> RelTime
 seconds x = convertMicrosecondsToRelTime (1000000 * x)
 
+-- A number of milliseconds in relative time.
```

Comment:
can we add some tests for this? maybe seems simple enough but I’d still prefer to err on the safe side.